extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -388,7 +399,7 @@ public function loadDataFromRevDataProvider() {
 				'id' => '23911114',
 				'name' => 'GarthTest800',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "GarthTest800" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)